src/pages/portfolio/index.tsx

```diff
@@ -1,11 +1,49 @@
 'use client';
 
+import React from 'react';
+import { XCircle } from 'lucide-react';
+import { Button } from '@penumbra-zone/ui/Button';
 import { Text } from '@penumbra-zone/ui/Text';
+import { Density } from '@penumbra-zone/ui/Density';
+import mobile from 'is-mobile';
 
 export const PortfolioPage = () => {
-  return (
-    <section>
-      <Text h2>Hi!</Text>
+  const isOnMobile = mobile();
+
+  return isOnMobile ? (
+    <section className='absolute inset-0 flex flex-col items-center justify-between p-4 gap-3 border-t  border-neutral-800'>
+      <div className='flex flex-col justify-center items-center p-0 gap-4 w-full flex-grow'>
```

Comment:
suggestion: can you please reuse the same block and apply it to the mobile sidebar (opened from header)? https://www.figma.com/design/xWsVprkEgEihweVN9isrGQ/DEX-(Incl.-UI-Lib)?node-id=3663-143558&node-type=frame&m=dev

this is not blocking but might be easier to do it right here than to open one more issue/pr

Reply:
I'm not sure what you mean here?